src/DynamoCore/Engine/EngineController.cs

```diff
@@ -38,6 +38,11 @@ public class EngineController : LogSourceBase, IAstNodeContainer, IDisposable
         /// </summary>
         public event AstBuiltEventHandler AstBuilt;
 
+        /// <summary>
+        /// This event is notify that the VMLibraries have been reset.
```

Comment:
`The event notifies clients that the VMLibraries have been reset.`
@aparajit-pratap is there any other info we want to add to this comment about this state - like - it is now safe to run code?

Reply:
Yes, we can add that the `VM is now _ready_ to run new code`.